client/src/modules/reports/generate/system_usage_stat/system_usage_stat.html

```diff
@@ -27,6 +27,7 @@ <h3 class="text-capitalize" translate>REPORT.SYSTEM_USAGE_STAT.TITLE</h3>
               max-date = "ReportConfigCtrl.today"
               date-value="ReportConfigCtrl.reportDetails.date"
               on-change="ReportConfigCtrl.onDateChange(date)"
+              limit-min-fiscal
```

Comment:
What does this do?

Reply:
Limits the minimum date to the first fiscal year date.